src/app/classes/run/run.ts

```diff
@@ -0,0 +1,17 @@
+import { Transition } from './transition';
+
+export class Run {
```

Comment:
Bei solchen reinen Datenklassen sollte man lieber mit Interfaces arbeiten.
Klassen haben immer einen ziemlichen Overhead und diese können bzw. sollten auch nicht in Templates genutzt werden.